src/main/java/uk/gov/hmcts/reform/sscs/config/CitizenCcdService.java

```diff
@@ -89,6 +90,22 @@ public SaveCaseResult createDraft(SscsCaseData caseData, IdamTokens idamTokens)
                 .build();
     }
 
+    public Optional<SscsCaseDetails> draftArchivedFirst(SscsCaseData caseData,
```

Comment:
I think we have removed the draft archived right.. now we are not archiving just creating the case from draft